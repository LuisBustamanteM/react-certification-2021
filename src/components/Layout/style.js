import styled from 'styled-components';

export const StyledLayout = styled.main`
  width: 100vw;
  height: 100vh;
`;