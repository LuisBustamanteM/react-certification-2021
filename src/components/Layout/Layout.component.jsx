import React from 'react';

import {StyledLayout} from './style';

function Layout({ children }) {
  return <StyledLayout>{children}</StyledLayout>;
}

export default Layout;
