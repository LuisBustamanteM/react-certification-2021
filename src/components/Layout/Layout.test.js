import React from 'react';

import Layout from './index';
import {render} from "@testing-library/react";

describe("Testing <Layout/>", () => {
    test("<Layout/> matches snapshot", () => {
        const component = render(<Layout />);
        expect (component.container).toMatchSnapshot();
    });

});