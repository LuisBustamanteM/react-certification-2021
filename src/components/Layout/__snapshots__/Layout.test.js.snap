// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Layout/> <Layout/> matches snapshot 1`] = `
<div>
  <main
    class="sc-bdfBQB dlQAfE"
  />
</div>
`;
